build.moxie

```diff
@@ -107,13 +107,13 @@ properties: {
   slf4j.version  : 1.7.12
   wicket.version : 1.4.22
   lucene.version : 5.5.2
-  jgit.version   : 4.1.1.201511131810-r
+  jgit.version   : 4.1.2.201602141800-r
   groovy.version : 2.4.4
   bouncycastle.version : 1.52
   selenium.version : 2.28.0
   wikitext.version : 1.4
-  sshd.version: 1.0.0
-  mina.version: 2.0.9
+  sshd.version: 1.6.0
```

Comment:
Apache SSHD 1.6.0 is still extremely old ([tagged June, 2017](https://github.com/apache/mina-sshd/releases/tag/sshd-1.6.0)) and has publicly documented security issues. The current release is 2.8.0.

It may be an upgrade you want to tackle separately, but it's likely worthwhile to do so. In particular, going to SSHD 2.8 will have [built-in support for rsa-sha2-256 and rsa-sha2-512](https://issues.apache.org/jira/browse/SSHD-1216) which is enabled by default. Since OpenSSH 8.8 disables ssh-rsa by default, this will save your end users a lot of hassle.

Reply:
Yeah when I sent this PR (in 2018) life was different. I tried to close this PR but failed. Good luck!